tests/utils/pod.go

```diff
@@ -77,19 +79,27 @@ func CreatePod(clientSet *kubernetes.Clientset, namespace string, podDef *k8sv1.
 }
 
 // DeletePodByName deletes the pod based on the passed in name from the passed in Namespace
-func DeletePodByName(clientSet *kubernetes.Clientset, podName, namespace string) error {
+func DeletePodByName(clientSet *kubernetes.Clientset, podName, namespace string, gracePeriod *int64) error {
```

Comment:
What's the difference between passing gracePeriod of int64(0) or nil here? nil means some default value?

Reply:
Yes, nil means use the default value.